services/system/AccountSys/ui/src/App.tsx

```diff
@@ -39,6 +43,23 @@ export interface AccountWithKey extends AccountWithAuth {
     privateKey: KeyPairWithAccounts["privateKey"];
 }
 
+/// Problem: All these exisitng system / user apps are using the old operations / mutations or whatever
+// Solution: Migrate them all to Plugin WASMs, and figure out
+// Importables! -> Providing a JS function to the WASM for it to call, to add actions to the actions array,
+// We don't solely operate on the return values of these WASM functions, we also require to monitor the functions that are called (that we give it via importables)
+//
+// Do we have operations that do network requests prior to providing actions to submit to chain?
+// JS Operations -> Can do network requests
+// WASM operations / functions -> Can't.
+//
+// Solution 1: Provide Rust with the option of network calls via importables
+// Solution 2: Figger it out in the app rather than operations... and dumb it down for the WASM
+//
+// In the future, we want these WASM functions to know context, e.g. who called this? and what app they were in when it was called?
```

Comment:
This will be done via an importable like, "getSender". The loader knows who called it because that information was sent by the supervisor, so it can fill in the getSender importable with the name of the caller service.